src/cascadia/CascadiaPackage/Package-Pre.appxmanifest

```diff
@@ -75,8 +75,43 @@
             Enabled="false"
             DisplayName="ms-resource:AppNamePre" />
         </uap5:Extension>
+        <uap3:Extension Category="windows.appExtension">
+            <uap3:AppExtension Name="com.microsoft.windows.console.host"
+                Id="OpenConsole-Dev"
+                DisplayName="OpenConsole"
+                Description="Console host built from microsoft/terminal open source repository"

```

Comment:
* [ ] This description and the one in Package.appxmanifest (stable) may need some minor fiddling in post.

Let's collect followups and roll them into another issue?

Reply:
What do you want these to say?

Reply:
No we aren't but I made them better anyway in the next rev.